core/include/mmcore/profiler/TimeMeasure.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * MegaMol
+ * Copyright (c) 2022, MegaMol Dev Team
```

Comment:
Please do never update any Copyright years and use always the original one. Reason: https://stackoverflow.com/a/2391555
(Exception would be if there is given a range 2000-2021, then the range should be updated. But a range is generally not required and we can omit this, to avoid needing to update it.)

Reply:
Ah, sorry, did not know about this and thanks for clarifying :)